extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -166,6 +221,10 @@ export class IframeTransport {
     }
     ampDoc.body.removeChild(frameData.frame);
     delete IframeTransport.crossDomainIframes_[type];
+    if (IframeTransport.performanceObservers_[this.type_]) {
+      IframeTransport.performanceObservers_[this.type_].disconnect();
+      IframeTransport.performanceObservers_[this.type_] = null;
```

Comment:
nit: `delete IframeTransport.performanceObservers_[type]`

Reply:
@keithwrightbos said to change from "delete" to "=null" 😆 Here is his comment: https://github.com/ampproject/amphtml/pull/11907#discussion_r149689970

I'll let the two of you discuss...

Reply:
LOL 😆 
I am also trying to be consistent with the 
```
delete IframeTransport.crossDomainIframes_[type];
```
pretty minor. I am Ok with both.

Reply:
Ok, I've changed line 223 to =null, for consistency. How's that?